src/rgw/rgw_op.cc

```diff
@@ -3416,6 +3416,11 @@ void RGWDeleteBucket::execute(optional_yield y)
     return;
   }
 
+  op_ret = rgw_remove_sse_s3_bucket_key(s);
+  if (op_ret != 0) {
+      // do nothing; it will already have been logged
```

Comment:
what benefit does the branch have, then?  should we cast the result to void or something?

Reply:
it documents that there is an error return, and that there's no sensible recovery at this point.  If the compiler cooperates, it may also present an opportunity to put a runtime breakpoint on it.  Conceivably future code could queue some sort of retry record.